bytecomp/bytelink.ml

```diff
@@ -501,6 +501,13 @@ let link_bytecode_as_c ppf tolink outfile =
 \n                    caml_sections, sizeof(caml_sections),\
 \n                    argv);\
 \n}\
+\nvalue caml_startup_exn(char ** argv)\
```

Comment:
`caml_startup_exn` should have been declared with `CAMLextern` above (see line 463).
I've fixed this in #71 this way:
```
-\nCAMLextern void caml_startup_code(\
-\n           code_t code, asize_t code_size,\
-\n           char *data, asize_t data_size,\
-\n           char *section_table, asize_t section_table_size,\
-\n           char **argv);\n";
+\n#include <caml/startup.h>\n";
```